Uno.Gallery/Uno.Gallery.Skia.Gtk/Properties/launchSettings.json

```diff
@@ -0,0 +1,10 @@
+{
+  "profiles": {
+	"Uno.Gallery.Skia.Gtk": {
+		"commandName": "Project",
+		"commandLineArgs": ""
+		//To navigate to a specific sample on startup
```

Comment:
Is this valid json? json infamously does not allow comments

Reply:
File Removed.